### ISSUE TITLE
reCAPTCHA API changed

### ISSUE THREAD

**Issue by kalaker** [claudehohl/Stikked#244]
There is now stuff to add in different areas of the webpage (before the end of </head> and </form>) and server-side stuff to do. The new API is more human-friendly. It in many cases can automatically detect you're human. If it doesn't, it gives you a captcha, but if it verifies you, it's a much easier process for users.
https://www.google.com/recaptcha/intro/index.html

**Comment by kalaker** [comment]
This will be helpful: https://github.com/google/ReCAPTCHA
As will: https://developers.google.com/recaptcha/docs/verify

**Comment by kalaker** [comment]
So basically, there is a whole new recommended library to use (linked to in last comment, the first URL), which you can almost use as drop-in replacement, if you modify `application/controllers/main.php`'s `_valid_recaptcha()` function to have `$resp` be equal to 
`$resp = verifyResponse($_POST["g-recaptcha-response"], $ra);`
However, then there is the issue that responses now come as a JSON response.

The old responses look something like (https://www.google.com/recaptcha/api/verify):
```
false
invalid-site-private-key
```

The new responses look something like (https://www.google.com/recaptcha/api/siteverify):
```json
{
  "success": false,
  "error-codes": [
    "missing-input-response",
    "missing-input-secret"
  ]
}
```

**Comment by insink71** [comment]
nice.  working example at https://paste.teamblueridge.org :)

**Comment by kalaker** [comment]
@insink71 : *nearly
I'll do a pull request once I get it fully up & running

**Comment by kalaker** [comment]
A pull request will be coming shortly.

**Comment by kalaker** [comment]
Fixed by pull request #245
